src/FSharp.Control.TaskSeq.SmokeTests/FSharp.Control.TaskSeq.SmokeTests.fsproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
```

Comment:
Not a fan of Directory.build.props? I guess a separate PR to put sln at top level with sln-level Directory.build.props that are common to build and test can come?

Reply:
Yeah. I don't like magic, too many things are then set in different locations or levels. I prefer being specific and having things in one place. But for `Version.props` it is required, otherwise the version plugins don't work.

Reply:
(if this ever gets into "many projects" territory, it may become a different story, but right now, I am fine with this)